src/components/Check/Check.scss

```diff
@@ -7,28 +7,35 @@
 // --------------------------------------------------
 // Check styles
 
-$checkBoxHeight: 20px;
+$checkBoxHeight: 22px;
 
 .ms-Check {
-  display: inline-block;
-  cursor: default;
-  vertical-align: top;
+  font-family: FabricMDL2Icons;
```

Comment:
Do we need to have font-family here and at line 62? Doesn't ms-Icon take care of that?